src/stores/Form.js

```diff
@@ -245,6 +247,7 @@ class FormStore {
     this.localizedData[l0][l1] = this.localizedData[l0][l1] || {};
 
     let assetMetadata = this.rootStore.client.utils.SafeTraverse(this.rootStore.assetMetadata, l0, l1) || {};
+    let otherMetadata = this.rootStore.client.utils.SafeTraverse(this.rootStore.assetMetadata, l0, l1) || {};
```

Comment:
Did you intend for this to be initialized from `this.rootStore.otherMetadata`?

Reply:
Yeah, I didn't want to combine it with assetMetadata and thought otherMetadata would suffice. Hm do you think searchablesMetadata needs its own variable in the rootStore?

Reply:
I mean here, assetMetadata and otherMetadata are set to identical values. is that intended?

Reply:
Oh no, otherMetadata should be set to rootStore.otherMetadata